.github/workflows/docs.yml

```diff
@@ -47,18 +47,7 @@ jobs:
         run: pnpm run docs:typedoc
 
       - name: 📚 Generate Markdown Docs
-        run: |
-          pnpm run docs:jsdoc --path packages/react-router/lib/components.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/hooks.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/dom/lib.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/dom/ssr/components.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/dom/ssr/server.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/dom-export/hydrated-router.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/dom/server.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/rsc/browser.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/rsc/server.rsc.ts --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/rsc/server.ssr.tsx --write
-          pnpm run docs:jsdoc --path packages/react-router/lib/rsc/html-stream/browser.ts --write
+        run: pnpm run docs:jsdoc --write
```

Comment:
We can now just run this for all files - only the ones marked `@public` get written out